src/integrationTest/resources/pubkey.asc

```diff
@@ -0,0 +1,97 @@
+-----BEGIN PGP PUBLIC KEY BLOCK-----
```

Comment:
we now have 4 keys in `resources`. it would be nice to put the in separate dirs so that it's clear which ones are used for what